model/src/main/java/org/cbioportal/model/SampleClinicalData.java

```diff
@@ -0,0 +1,25 @@
+package org.cbioportal.model;
+
+import org.cbioportal.model.summary.ClinicalDataSummary;
+
+public class SampleClinicalData extends ClinicalDataSummary {
+
+    private Sample sample;
+    private ClinicalAttribute clinicalAttribute;
```

Comment:
This non-Summary model classes are for DETAILED projection, and the default projection is SUMMARY, so it will be duplicated only if users want to, they will probably use DETAILED projection in some special cases. But it's good to have that option.